lists/python/test_linked_list.py

```diff
@@ -46,7 +46,7 @@ def test_ll_add_to_populated_list():
     actual = subject.add(new_node)
 
     # assert
-    assert actual == 'new'
+    assert actual == new_node
```

Comment:
This is a good first assert. The real value in these asserts will actually be in asserting the `next` and `prev` nodes are what we expect them to be. It would be simple to pass these tests by implementing the add function as shown below, but that's not really a linked list is it.

```python
def add(self, node):
    return node
```